plugins/stylescombo/plugin.js

```diff
@@ -115,8 +139,10 @@
 							elements = elementPath.elements,
 							pathDepth = elements.length;
 
-						if (elementPath.elements[0].getName() === 'li') {
-							pathDepth = 2;
+						var cutOffIndex = getCutOffIndex(elements, ['ol', 'ul']);
```

Comment:
We're not necessarily walking the selection though, we want the entire element path that leads to the element the current selection is in.  

contrived example:

```
<ol><li><p>my selection is here<b> some random stuff</b></p></li></ol>
```

The abilty to get `[p, li, ol]`